test/factories/item_attachments.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :item_attachment do
-    item_id { nil }
-    kind { "" }
+    kind { "manual" }
     other_kind { "MyString" }
+    file { Rack::Test::UploadedFile.new(Rails.root.join("test", "fixtures", "files", "tool-image.jpg"), "image/jpg") }
```

Comment:
This is a good addition! The factory-generated item attachments will be a lot more realistic now.